README.md

```diff
@@ -1,3 +1,36 @@
-This is a nascent project for implementing a rich text layout engine backed
-by the [swash](https://github.com/dfrg/swash) crate and implementing the 
-[piet](https://github.com/linebender/piet) text API.
+<div align="center">
+
+# Parley
+
+**Rich text layout**
+
+[![Latest published parley version.](https://img.shields.io/crates/v/parley.svg)](https://crates.io/crates/parley)
+[![Documentation build status.](https://img.shields.io/docsrs/parley.svg)](https://docs.rs/parley)
+[![Dependency staleness status.](https://deps.rs/repo/github/linebender/parley/status.svg)](https://deps.rs/repo/github/linebender/parley)
+[![Linebender Zulip chat.](https://img.shields.io/badge/Linebender-%23gpu-blue?logo=Zulip)](https://xi.zulipchat.com/#narrow/stream/205635-text)
```

Comment:
In Vello, we have put this at the start of the list of badges. It's probably best to have a standard order

Also, the text says `#gpu`, when the link is to `#text`

Reply:
> Also, the text says `#gpu`, when the link is to `#text`

Good catch!

> It's probably best to have a standard order

We should absolutely have a standard set of badges and in the same order.

I think all the badges should provide some value. So for example I didn't add the CI badge here, because our merge queue doesn't merge anything with a failing CI, so there's no value in that badge.

For the remaining badges, my take was that knowing the latest crates.io version (or the docs link) is likely to be of interest to more people than a link to our Zulip.

I'm willing to bikeshed a bit on this and don't feel too strongly about any specific order as long as we don't have an obnoxious number of these badges.

Vello doesn't yet have the crates.io badge at all, so the set of badges is different. I would like to agree upon a set and order here, and then I can send the amendments to all other repos.

So when thinking about establishing a new standard set and order, what do you think? Should the Zulip still appear before the crates.io badge for example?

Reply:
I don't think deciding on this order should block this PR. Maybe it's worth having a Zulip thread?
I think that the Zulip is a key advantage of our community, and I am inclined to put it first for that reason.

In Android Trace, for the crates, I split them into two lines:
Zulip | License | Dependencies (of published version) | crate compatibility (such as wgpu=0.19)
crates.io | docs | ci

I think the CI badge is useful for specifically crates.io/docs.rs, because it shows that we do have CI running. (Although I haven't actually applied that in Android Trace).
It isn't that useful for a GitHub Readme, but sharing the same readme across both is worth that small cost

Reply:
Yeah let's discuss this in Zulip later. I'll create a thread after office hours or so. For now I'll just merge this as-is and we can amend later.